chat-widget/automation_tests/yarn.lock

```diff
@@ -0,0 +1,6298 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why are there 2 yarn.lock files modified here? Can we revert the changes for this one "/chat-widget/yarn.lock"?

Reply:
Oops. seems yarn.lock modified while running yarn command. I will revert '/chat-widget/yarn.lock' file.